ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/copyfrom/oomph/P2Index.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2014, 2016 Eike Stepper (Loehne, Germany) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *    Eike Stepper - initial API and implementation
+ */
+package org.eclipse.pde.internal.core.copyfrom.oomph;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.equinox.p2.metadata.Version;
+
+/**
+ * @author Eike Stepper
+ */
+public interface P2Index {
```

Comment:
I don'T think we need the interface abstraction here, merge with the impllmentaiton